Neos.Media/Migrations/Mysql/Version20181104152203.php

```diff
@@ -0,0 +1,57 @@
+<?php
+namespace Neos\Flow\Persistence\Doctrine\Migrations;
+
+/*
+ * This file is part of the Neos.Media package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Doctrine\DBAL\DBALException;
+use Doctrine\DBAL\Migrations\AbortMigrationException;
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
```

Comment:
This is breaking 4.2.0 - there it still is `Doctrine\DBAL\Migrations\AbstractMigration`

Reply:
I correct myself, the reason are wrong dependencies in Flow dev collection vs standalone, caused by dependabot. see https://github.com/neos/flow-development-collection/pull/1475 and https://github.com/neos/flow-development-collection/pull/1476